src/System.IO.Pipelines/src/System/IO/Pipelines/ResultFlags.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.IO.Pipelines
+{
+    [Flags]
+    internal enum ResultFlags : byte
+    {
+        None = 0,
+        Canceled = 1,
```

Comment:
Nit: For [Flags] enums, it's usually clearer to use hex notation for the values.